src/Rules/Functions/FactoriesFunctionArgumentTypeRule.php

```diff
@@ -31,8 +31,7 @@
 final class FactoriesFunctionArgumentTypeRule implements Rule
 {
     /**
-     * @var array<string, string>
-     * @phpstan-var array<string, class-string>
+     * @var array<string, class-string>
```

Comment:
Is `class-string` now valid PHPDoc? Or are you just switching to native everywhere?

Reply:
Now valid with intelephense v1.10